crates/astria-conductor/src/celestia/fetch.rs

```diff
@@ -119,7 +119,7 @@ async fn fetch_blobs_with_retry(
         async move {
             match client.blob_get_all(height, &[namespace]).await {
                 Ok(blobs) => Ok(blobs),
-                Err(err) if is_blob_not_found(&err) => Ok(vec![]),
+                Err(err) if is_null_blobs(&err) => Ok(vec![]),
```

Comment:
I think we should keep the old check as well, so that celestia upgrade can happen seamlessly without requiring conductor upgrade in lockstep.